### ISSUE TITLE
Build error after update to Angular 14

### ISSUE THREAD

**Issue by symonfly82** [single-spa/single-spa-angular#440]
### Demonstration
After updating in my angular project all angular packages to v 14, I receive the below error:
 
Generating browser application bundles (phase: setup)...
[0] An unhandled exception occurred: Cannot find module '@angular/cli/utilities/find-up'
[0] Require stack:
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\single-spa-angular\lib\webpack\index.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-builders\custom-webpack\dist\utils.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-builders\custom-webpack\dist\custom-webpack-builder.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-builders\custom-webpack\dist\transform-factories.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-builders\custom-webpack\dist\generic-browser-builder.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-builders\custom-webpack\dist\dev-server\index.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-devkit\architect\node\node-modules-architect-host.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular-devkit\architect\node\index.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\src\command-builder\architect-base-command-module.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\src\command-builder\architect-command-module.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\src\commands\build\cli.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\src\command-builder\command-runner.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\lib\cli\index.js
[0] - C:\Projects\GitHub\is.selfnet.clientapp\node_modules\@angular\cli\lib\init.js

I think could be simply a reference changed problem

### Expected Behavior
Build completes succesfully

### Actual Behavior
Build error

**Comment by ALP011216** [comment]
https://github.com/single-spa/single-spa-angular/blob/9e8793fea12cc3778b90ae2ca21bff22c949d58c/libs/single-spa-angular/webpack/index.ts#L2
Should be changed to
```ts
import { findUp } from '@angular/cli/src/utilities/find-up'; 
```

**Comment by ALP011216** [comment]
https://github.com/single-spa/single-spa-angular/blob/9e8793fea12cc3778b90ae2ca21bff22c949d58c/libs/single-spa-angular/webpack/index.ts#L2

Utilities are now in the `src` directory

**Comment by kfrederix** [comment]
Ran into this as well. As a quick workaround, we adapted the path in our `node_modules` and used [patch-package](https://www.npmjs.com/package/patch-package) to generate the following patch:

```
diff --git a/node_modules/single-spa-angular/lib/webpack/index.js b/node_modules/single-spa-angular/lib/webpack/index.js
index 563a46a..02a4d61 100644
--- a/node_modules/single-spa-angular/lib/webpack/index.js
+++ b/node_modules/single-spa-angular/lib/webpack/index.js
@@ -1,7 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 const fs = require("fs");
-const find_up_1 = require("@angular/cli/utilities/find-up");
+const find_up_1 = require("@angular/cli/src/utilities/find-up");
 const externals_1 = require("./externals");
 const remove_mini_css_extract_1 = require("./webpack-5/remove-mini-css-extract");
 const defaultExtraOptions = {
```

If will try to find some time this week to make a PR here to update to angular 14 and fix this properly.

**Comment by arturovt** [comment]
I've published `single-spa-angular@7.0.0`. See what has been changed: https://github.com/single-spa/single-spa-angular/releases/tag/v7.0.0.

I'll close the issue for now, feel free to re-open if you face any issues.
